packages/lib-user/src/components/shared/AllProjects/README.md

```diff
@@ -0,0 +1,43 @@
+# AllProjects
+
+## UI Decision
+
+The components in this folder build a classic pagination UI. Projects get filtered out of a page when a user did not make classifications (`/project_preferences`) or a project is private (`/projects`). This means the UI is designed for `PAGE_SIZE`, but a couple of ProjectCards might be missing even though Pagination hasn't reached the last page. This is an intentional UI tradeoff.
+
+An infinite scroll UI with a 'Load More' button and `useSWRInfinite` was considered, but panoptes and ERAS API limitations prevent that strategy. ERAS does not have pagination capabilities. Panoptes `/api/projects` can be passed a string of multiple project ids, but if the query's `page_size` is smaller than the number of project ids, the `projects` response is in an unpredictable order. The response does not respect the order of project ids passed in the query.
```

Comment:
Please read for decision about classic pagination despite the chance a "page" might not always have exactly 20 project cards.